### PR TITLE
hw7 done

### PR DESCRIPTION
**Pull request by vitaholovach**

### CODE REVIEW

File: work7/task7_2.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+def fun(variable):
+    letters = ['a', 'e', 'i', 'o', 'u']
+    if (variable in letters):
+        return True
+    else:
+        return False
+sequence = ['g', 'e', 'e', 'j', 'k', 's', 'p', 'r']
+filtered = filter(fun, sequence)
```
Comment (bohdandovbsyh) [inline work7/task7_2.py]: looks like you use default filter. You need t implement your own one

File: work7/task7_3.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+def square(num):
+    return num ** 2
+
+numbers = [1, 2, 3, 4, 5]
+squares = list(map(square, numbers))
```
Comment (bohdandovbsyh) [inline work7/task7_3.py]: you use default map. I want you to implement your own one

File: work7/task7_4.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+def max_min(data):
+  a = data[0]
+  b = data[0]
```
Comment (bohdandovbsyh) [inline work7/task7_4.py]: What is a and b?

File: work7/task7_2.py (hunk 1 above)
Comment (vitaholovach) [reply work7/task7_2.py]: fix

File: work7/task7_3.py (hunk 2 above)
Comment (vitaholovach) [reply work7/task7_3.py]: fix

File: work7/task7_4.py (hunk 3 above)
Comment (vitaholovach) [reply work7/task7_4.py]: fix

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**